crates/nu-cli/src/config_files.rs

```diff
@@ -92,7 +96,7 @@ pub(crate) fn read_config_file(
                 .read_line(&mut answer)
                 .expect("Failed to read user input");
 
-            let config_file = include_str!("../docs/sample_config/default_config.nu");
+            let config_file = include_str!("../../../docs/sample_config/default_config.nu");
```

Comment:
I'm slightly dubious here, but it seems to be correct with regards to how it worked previously.

Reply:
Yeah, include_str feels a bit magic, but I *think* so long as it works we're... okay?

Reply:
It seems to work but I'm unsure how it works once nushell is distributed via something like homebrew? I don't fully understand how that works though... If that's fine then it solves a headache I have in couchbase shell too :)

Reply:
Right, but is the path it refers to always valid?

Reply:
yeah, since it just has to be be valid at build time, and it's relative to that subcrate.